exercises/node-javascript/multi-cmk-start/store.js

```diff
@@ -0,0 +1,83 @@
+const {
```

Comment:
@mattsb42-aws @lizroth Here is the start store file.  Both CMKs need to be added.